### ISSUE TITLE
`v fmt .` Introduces a V error when []thread struct{} is in a module.

### ISSUE THREAD

**Issue by charliemikels** [vlang/v#10768]
**V version:** V 0.2.2 eb96ad1.ba9b53c
**OS:** linux, "Manjaro Linux"

When using `[]threads struct{}` inside of a module, `v fmt .` unnecessarily adds the module name to the struct. (eg: `[]threads mod_name.struct{}`.) This causes a V error when running the main program, because the module can't find a sub module with it's name. 

<details>
<summary>Click to expand the example case:</summary>

Directory tree:
```
.
├── main.v
├── my_module
│   └── my_module.v
└── v.mod

1 directory, 3 files

```
main.v:
```V
import my_module as mm

fn main() {
	things := mm.do_something()
	println('number of things: $things.len')
}
```

my_module/my_module.v:
```V
module my_module

pub struct MyStruct {
	thing1 string
	thing2 int
}

fn do_one_thing(num int) MyStruct {
	return MyStruct{
		thing1: 'the number $num'
		thing2: num * num
	}
}

pub fn do_something() []MyStruct {
	numbers := [1, 2, 3, 4, 5]
	mut threads := []thread MyStruct{}
	for n in numbers {
		threads << go do_one_thing(n)
	}
	the_structs := threads.wait()
	return the_structs
}
```

initial `v run .`:
```
number of things: 5
```

`v fmt . -w` output:
```
Reformatted file: /path/to/my_module/my_module.v
Already formatted file: /path/to/main.v
```

`v run .` again:
```
my_module/my_module.v:17:26: error: unknown module `my_module`
   15 | pub fn do_something() []MyStruct {
   16 |     numbers := [1, 2, 3, 4, 5]
   17 |     mut threads := []thread my_module.MyStruct{}
      |                             ~~~~~~~~~
   18 |     for n in numbers {
   19 |         threads << go do_one_thing(n)
```

`git diff` output:
```
diff --git a/my_module/my_module.v b/my_module/my_module.v
index 348f964..698c6ec 100644
--- a/my_module/my_module.v
+++ b/my_module/my_module.v
@@ -14,7 +14,7 @@ fn do_one_thing(num int) MyStruct {
 
 pub fn do_something() []MyStruct {
 	numbers := [1, 2, 3, 4, 5]
-	mut threads := []thread MyStruct{}
+	mut threads := []thread my_module.MyStruct{}
 	for n in numbers {
 		threads << go do_one_thing(n)
 	}

```
</details>

**Comment by spytheman** [comment]
I can also reproduce it with `v fmt -diff my_module/my_module.v`.

**Comment by spytheman** [comment]
Fixed in V 974737f.
